app/views/posts/show.html.erb

```diff
@@ -13,5 +13,10 @@
       <p> <%= comment.user.name %> : <%= comment.text %><p>
     <% end %>
     </div>
-  
+
+<%= form_with model: @like, url: likes_path, method: :post, local: false do |f| %>
+<%= hidden_field(:post, :id) %>
+<%= f.submit "Add like", class: "btn btn-primary" %>
+<% end %>
+
 </div>
```

Comment:
- [ ] [OPTIONAL] When I try to create a post. I am getting this error. I am marking this as optional since it was not mentioned by the previous reviewer.

![image](https://github.com/francksefu/blog_web/assets/86318284/e000c9c6-4d55-42d8-b9ea-6a34b6b0632d)

**I am therefore not able to confirm that creating the post, comment and likes is possible.**